### DISCUSSION TITLE
Workaround for this?

### DISCUSSION THREAD

**Discussion by mschwartz** [vercel/next.js#42391] [Help]
I am running a Next JS app in an iframe in a PWA.  The iframe is shown when a certain tab is selected in the PWA's user interface.

When switching tabs very fast, I occasionally see this error in the console.  I'm not sure if this can be reproduced without the iframe, simply hitting reload over and over very fast might do it.

![2022-11-02 at 5 27 PM](https://user-images.githubusercontent.com/329637/199626996-f4625902-e174-4e38-9391-b9e1429791b7.png)

**Comment by mschwartz** [comment]
I have no _middleware type files.

**Comment by icyJoseph** [comment]
Does the iframe have the `allow-top-navigation` attribute setup? I reckon that, although you have no middleware setup, there's a call to check if there's one, and that's perhaps hitting an API that's not available to iframes.

**Comment by mschwartz** [comment]
The reason I am using an iframe is so I can control the document within, like looking at the url being viewed.  In the PWA context, there is no browser chrome to navigate the iframe content - no home, back, reload buttons or address bar - I have to do all that in the PWA’s UI…

I don’t have any of the sandbox type attributes set.  Just iframe with src prop.

I can see the iframe starts loading the next js rendered content but still at the blank screen stage.

it may be a warning I can ignore, since I don’t expect to be switching tabs so quickly.  A good Q/A person might find it though.

it seems like an edge case kind of a bug in next js, though.  Probably should be fixed.

I looked at the issues and there already was one about this, with a PR marking it fixed…
